docs/FAQ.md

```diff
@@ -64,3 +64,31 @@ a local machine or foreign region)—I/O is much faster.
 
 1. Increase [`QUILT_TRANSFER_MAX_CONCURRENCY`](api-reference/cli.md#quilt_transfer_max_concurrency)
 above its default to match your available vCPUs.
+
+## Does Quilt work with R?
+
+In the scientific computing community, the [R Project](https://www.r-project.org/)
+is commonly used as an alternative, or companion, to Python. It is a language and
+environment for statistical computing and graphics, and is available as Free Software
+under the [GNU General Public License](https://www.r-project.org/COPYING).
+
+Currently there are no plans to release a Quilt package for distribution through
+the [CRAN package repository](https://cloud.r-project.org/). However, you can still
+use Quilt with R, using either:
+
+1. The Command Line Interface (CLI) API
+1. [Reticulate](https://rstudio.github.io/reticulate/)
+
+### Using the Quilt CLI API with R
+You can script the Quilt CLI directly from your shell environment and chain it
+with your R scripts to create a unified workflow:
+
+```bash
+quilt3 install
+[Run R commands or scripts]
+quilt3 push
+```
```

Comment:
I find it hard to understand follow what  is happening without context. Perhaps:
```
$ quilt3 install # download package
[Run R commands or scripts] # modify or add files
$ quilt3 push # upload package
```
Unless there's a different convention that works better?

Reply:
I'm not sure you can isolate this to just a CLI (i.e. the $ CLI prefix). You can have a bash script that does all this in one executable

Reply:
I followed the convention used earlier in the page

Reply:
I've added more context and resubmitted